### PR TITLE
Prepare a 0.6.2 release.

### PR DESCRIPTION
**Pull request by ltratt**

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -1,3 +1,11 @@
+# grmtools 0.6.2 (2020-03-22)
+
+* Fix two two Clippy warnings and suppress two others.
```
Comment (ptersilie) [inline CHANGES.md]: Remove one `two`.

Comment (ltratt) [reply CHANGES.md]: Oops, well that's a very meta-typo! Fixed in https://github.com/softdevteam/grmtools/pull/173/commits/d80a0ea544829c5664703ca15ed5336a0ba7d6f0.

**Comment by ptersilie** [comment]
Please squash.

**Comment by ltratt** [comment]
Squashed.

**Comment by ptersilie** [comment]
Bors r+

**Pull request merged**